src/app/pages/auth/LoginFormContainer.jsx

```diff
@@ -3,15 +3,8 @@ import { compose } from 'redux'
 import { reduxForm } from 'redux-form'
 import LoginForm from './LoginForm'
 import { connectFormResource } from 'common/utils/resource'
-import { connect } from 'react-redux'
-import navigate from 'common/router/navigate'
 
 class LoginFormContainer extends PureComponent {
-  componentWillReceiveProps({ submitting, submitSucceeded, navigate }) {
-    if(!submitting && submitSucceeded) {
-      navigate('dashboard')
-    }
-  }
```

Comment:
can you add example how to navigate after submit ?

Reply:
1. this code is unnecessary here because of check access